components/bl-collapsiblePanel-component/src/index.js

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useState } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function CollapsiblePanelComponent({ component, elRef, eventHandlers, pods }) {
+  const { classList, style, display, title } = component;
+  const { onOpen, onClose } = eventHandlers;
+
+  const [isActive, setIsActive] = useState(false);
+  const [podClass, setPodClass] = useState('');
```

Comment:
instead of managing an extra hook, you can just calculate it in runtime

```
cn('bl-customComponent-collapsiblePanel', ...classList, {'panel-expanded':expanded})
...


and then in styles

.bl-customComponent-collapsiblePanel{
 &.panel-expanded{
    .panel-title { ... }
    .panel-content{ ... }
  }
}
```